tests/jinja2/Jinja2BearTest.py

```diff
@@ -23,6 +23,25 @@
                    r'{{  var }}'),
     settings={'variable_spacing': '0'})
 
+Jinja2BearStatementSpacingTest = verify_local_bear(
+    Jinja2Bear,
+    valid_files=(r'{% statement1 %}',
+                 r'{% statement2 foobar %}'),
+    invalid_files=(r'{%foo bar %}',
+                   r'{% foo bar%}'
+                   r'{%foo bar%}'
+                   r'{%   foo bar  %}'))
+
+Jinja2BearCustomStatementSpacingTest = verify_local_bear(
+    Jinja2Bear,
+    valid_files=(r'{%statement1%}',
+                 r'{%statement2 foobar%}'),
+    invalid_files=(r'{%foo bar %}',
+                   r'{% foo bar%}'
+                   r'{% foo bar %}'
+                   r'{%   foo bar  %}'),
+    settings={'statement_spacing': '0'})
```

Comment:
I think you can put this zero without single quotes.

Reply:
@khanchi97 That's true but I don't think it creates much of a difference here. Other similar tests have 0 within single quotes.